docs/_docs/framework/capabilities/optimize/workloads.tobedeleted

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+grand_parent: FinOps Framework
+parent: Optimize
+title: Workload optimization
+permalink: /framework/capabilities/optimize/workloads
+nav_order: 2
+description: This article helps you understand the workload optimization capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+author: bandersmsft
+ms.author: banders
+ms.date: 06/23/2023
+ms.topic: conceptual
+ms.service: finops
+ms.reviewer: micflan
+---
+
+<span class="fs-9 d-block mb-4">Workload optimization</span>
+This article helps you understand the workload optimization capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+  <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [▶️ Getting started](#️-getting-started)
+- [🏗️ Building on the basics](#️-building-on-the-basics)
+- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
+- [⏩ Next steps](#-next-steps)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+<a name="definition"></a>
+**Workload optimization capability involves the analysis and fine-tuning of cloud resources to align with unique usage patterns, prioritizing efficiency and ensuring that workloads deliver optimal business value relative to their cost.**
+{: .fs-6 .fw-300 }
+
+Review how services are being used and ensure each is maximizing return on investment. Evaluate and implement best practices and recommendations.
+
+Every cost should have direct or indirect traceability back to business value. Eliminate fully "optimized" resources that aren't contributing to business value.
+
+Review resource usage patterns and determine if they can be scaled down or even shutdown (to stop billing) during off-peak hours. Consider cheaper alternatives to reduce costs.
+
+An effective workload optimization plan can significantly reduce costs by adjusting configuration to match supply to demand dynamically, ensuring the most effective utilization.
+
+<br>
+
+## ▶️ Getting started
+
+When you first start working with a service and managing costs in the cloud, prioritize leveraging native tools within the portal to drive efficiency and optimize costs.
+
```

Comment:
Should we mention to use the Advisor Cost Optimization Workbook in this section? 

Use and customize the [Cost optimization workbook](https://learn.microsoft.com/en-us/cloud-computing/finops/cost-optimization-workbook). The Cost Optimization workbook is a central point for some of the most often used tools that can help achieve utilization and efficiency goals.